cinch/roles/certificate_authority/tasks/main.yml

```diff
@@ -1,7 +1,20 @@
-- name: install certificate authority
+- name: install certificate authority from URL
   get_url:
     url: "{{ item }}"
     dest: /etc/pki/ca-trust/source/anchors/
+    owner: root
+    group: root
+    mode: 0644
     validate_certs: "{{ certificate_authority_validate_certs }}"
   with_items: "{{ certificate_authority_urls }}"
   notify: install certificate authority
```

Comment:
That's a good point, I don't see we would use become in any other roles, for example when installing packages with yum, where you also need root. I'd say @robled is right and it's unnecessary...